content/docs/session.md

```diff
@@ -30,13 +30,48 @@ You can also load a session file using the [`-openSession` command line argument
+    - If you want Notepad++ to automatically save your custom session whenever you exit Notepad++, so that the session file will always remember which files have been closed or opened, then you need to be in **`☑ Open session in a new instance (and save session automatically on exit)`** mode.
+- The **[Settings > Preferences > Backup](../preferences/#backup) > `☐ Remember current session for next launch`** must be _off_ (because, if that option is on, then only one Notepad++ instance gets to write a session file when it's closed, so your custom session's instance might not be the one to save changes)
+- Pick a unique file extension to always use for Notepad++ sessions (for example, `.nps` for "**N**ote**p**ad++ **S**ession").  There are then two possible ways to automatically associate those with Notepad++:
+    1. You can set **[Settings > Preferences > MISC](../preferences/#misc) > Session file ext. `____`** to your extension (like `nps`, without the leading `.` dot) inside Notepad++; and from Windows Explorer, right-click on a session file and tell it to _always_ **Open With** Notepad++.
+        - A side effect of this method is that if you use Notepad++'s **File > Open** (or equivalent) on a file with the same extension, it will treat it as a session file, and open the files from that session rather than letting you edit the underlying XML text of that session file.  Thus, if you _want_ to be able to edit the session-file XML manually in Notepad++, do _not_ use this method.
+    2. You can set up a new file association in the Windows registry, which will associate your chosen extension (using `nps` in this example) with Notepad++, calling it with the [command line options](../command-prompt) `-openSession -multiInst` to make sure that it opens the file as a session file
+        - For this method, you do _not_ need to set the MISC preference from the first option.
+        - Here is an example `nps_as_session_file.reg` registry file, which you can run from Windows Explorer:
+            ```
+            Windows Registry Editor Version 5.00
+
+            [HKEY_CURRENT_USER\SOFTWARE\Classes\.nps]
+            @="Notepad++_session_file"
+
+            [HKEY_CURRENT_USER\SOFTWARE\Classes\Notepad++_session_file]
+
+            [HKEY_CURRENT_USER\SOFTWARE\Classes\Notepad++_session_file\shell]
+
+            [HKEY_CURRENT_USER\SOFTWARE\Classes\Notepad++_session_file\shell\open]
+
+            [HKEY_CURRENT_USER\SOFTWARE\Classes\Notepad++_session_file\shell\open\command]
+            @="\"C:\\Program Files\\Notepad++\\notepad++.exe\" -openSession -multiInst \"%1\""
+            ```
+            _Note_: If you used a different extension than `.nps`, you need to edit this file to use your extension, before running it.
```

Comment:
Maybe one more info for the used note:

"Note: If you used a different extension than .nps, you need to edit this file to use your extension, before running it and if you want this setting to be system-wide (and not for the current user only), you have to replace the HKEY_CURRENT_USER with HKEY_LOCAL_MACHINE."

Reply:
Updated.